src/test/java/cc/protea/spreedly/SpreedlyTest.java

```diff
@@ -28,6 +32,58 @@ public void testGetGatewayProviders() throws Exception {
     	Assert.assertTrue(! list.isEmpty());
     }
 
+	@Test
+	public void testCreate() throws Exception {
+		if (spreedly == null) {
+			return;
+		}
+		SpreedlyGatewayAccount account = new SpreedlyGatewayAccount();
+		account.gatewayType = "sage";
+		SpreedlyGatewayCredential c = new SpreedlyGatewayCredential();
+		c.name = "merchant_id_number";
+		c.value = "12345";
+		account.credentials.add(c);
+
+		final SpreedlyGatewayAccount createdAccount = spreedly.create(account);
+		Assert.assertNotNull(createdAccount);
+		Assert.assertTrue(createdAccount.getToken() == "sage");
+		Assert.assertTrue(createdAccount.getGatewayType() == "sage");
+		Assert.assertTrue(!createdAccount.getCredentials().isEmpty());
+		Assert.assertTrue(createdAccount.getCredentials().get(0).getName() == "merchant_id_number");
+		Assert.assertTrue(createdAccount.getCredentials().get(0).getValue() == "12345");
+
+		testAccount = createdAccount;
+	}
+
+	@Test
+	public void testRedact() throws Exception {
+		if (spreedly == null) {
+			return;
+		}
```

Comment:
Am I correct that this test will still read as a passing test if (spreedly == null) is true?